content/core-features/occupancy.textile

```diff
@@ -0,0 +1,17 @@
+---
+title: Occupancy
+section: core-features
+index: 29
+meta_description: "The occupancy feature returns metrics relating to clients connected to a channel."
+meta_keywords: "occupancy, metrics, occupancy metrics"
+---
+
+The occupancy feature enables you to view the number of occupants connected to a channel and specific metrics about their connections.
+
+The metrics returned by the occupancy feature include the number of clients connected to a channel, the number of those authorized to subscribe to messages on the channel and the number of those clients authorized to publish to the channel. The occupancy metrics also include the number of connections authorized to subscribe to presence events, the number authorized to publish presence events and the number of connections currently entered into "presence":/core-features/presence on the channel.
```

Comment:
"the number of connections currently entered": this is not strictly accurate, as a single connection may enter multiple presence members. I would prefer that this explicitly says the number of presence members currently present.

Reply:
Ah really good point, thanks. Updated in https://github.com/ably/docs/pull/1232